server/src/internalClusterTest/java/org/elasticsearch/snapshots/ConcurrentSnapshotsIT.java

```diff
@@ -644,8 +644,8 @@ public void testQueuedOperationsOnMasterDisconnect() throws Exception {
         networkDisruption.stopDisrupting();
 
         logger.info("--> make sure all failing requests get a response");
-        expectThrows(RepositoryException.class, firstDeleteFuture::actionGet);
-        expectThrows(RepositoryException.class, secondDeleteFuture::actionGet);
+        assertAcked(firstDeleteFuture.actionGet());
```

Comment:
Can we do normal `.get()` here just to not lose the stacktrace in case of a failure?